tests/eigenwave3d.py

```diff
@@ -156,14 +157,14 @@ def default(compiler=None, execute=False, nthreads=1,
         f.write(str(tile))
         f.close()
     domain_size = (1.0, 1.0, 1.0)
-    grid_size = (100, 100, 100)
+    grid_size = (247, 247, 247)
```

Comment:
For production runs this can be changed locally but I don't think this change should be committed for the test case. If this really needs to be changed then (a) update the comment above and (b) adjust the value of 'dt' such that the Courant number remains the same.